samples/nrf9160/serial_lte_modem/src/slm_at_icmp.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef SLM_AT_ICMP_
+#define SLM_AT_ICMP_
+
+/**@file slm_at_icmp.h
+ *
+ * @brief Vendor-specific AT command for ICMP service.
+ * @{
+ */
+
+#include <zephyr/types.h>
+#include "slm_at_host.h"
```

Comment:
You can see that in the .c file, there is a need to include slm_at_host.h and currently it's done via slm_at_icmp.h. But if needed I can change to all three blocks (TCPIP, GPS, ICMP) following your suggestion.

Reply:
While not a huge harm in this particular case, I prefer when headers include other headers only when they have direct dependencies to their content (like some types used in function declaration). Otherwise, it's easy to end-up with an include-spaghetti. If `slm_at_host.h` is needed by the `.c` file, it should be included directly in there.

Reply:
Thanks, will be updated in next commit of new functionality

Reply:
Still not adressed